.github/workflows/python-app.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Lint with flake8
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
this is ok for now... but in general is best to store these options into a config file.

In this way, when you run flake8 outside of GitHub action you will get similar results.

In general we want to have as few things inside github actions yaml file so that what we run locally is the same to what we run for GHA (GitHub Actions).

My suggestion is to create a setup.cfg file and add the conffig there
 https://flake8.pycqa.org/en/latest/user/configuration.html

Reply:
Don't forget to remove the command line configs

Reply:
Sorry I missed the "remove the command line configs"; will do it in next PR.